src/UnityExtension/Assets/Editor/GitHub.Unity/UI/Window.cs

```diff
@@ -60,14 +60,20 @@ public static void GitHub_CommandLine()
             EntryPoint.ApplicationManager.ProcessManager.RunCommandLineWindow(NPath.CurrentDirectory);
         }
 
-#if DEBUG 
-        [MenuItem("GitHub/Select Window")] 
-        public static void GitHub_SelectWindow() 
-        { 
-            var window = Resources.FindObjectsOfTypeAll(typeof(Window)).FirstOrDefault() as Window; 
-            Selection.activeObject = window; 
-        } 
-#endif 
+#if DEBUG
+        [MenuItem("GitHub/Select Window")]
+        public static void GitHub_SelectWindow()
+        {
+            var window = Resources.FindObjectsOfTypeAll(typeof(Window)).FirstOrDefault() as Window;
+            Selection.activeObject = window;
+        }
+
+        [MenuItem("GitHub/Restart")]
+        public static void GitHub_Restart()
+        {
+            EntryPoint.Restart();
```

Comment:
Restart without a domain restart?

Reply:
Yes. It's going to leak but it's only for our use anyway.